clients/banking/src/components/CardWizardIndividualDelivery.tsx

```diff
@@ -39,142 +39,182 @@ type CardIndividualDeliveryConfig = {
 
 type Props = {
   members: Member[];
-  address: CompleteAddressWithContactInput;
+  address?: CompleteAddressWithContactInput;
   onSubmit: (cardDeliveryConfig: CardIndividualDeliveryConfig) => void;
 };
 
 export type CardWizardIndividualDeliveryRef = { submit: () => void };
 
-export const CardWizardIndividualDelivery = forwardRef<CardWizardIndividualDeliveryRef, Props>(
-  ({ members, address, onSubmit }: Props, ref) => {
-    const [currentCardIndividualDeliveryConfig, setCardIndividualDeliveryConfig] =
-      useState<CardIndividualDeliveryConfig>(() => members.map(member => ({ member, address })));
-
-    const [editingAddress, setEditingAddress] = useState<[Address, number] | null>(null);
-
-    const hasSomeError = currentCardIndividualDeliveryConfig.some(config =>
-      isNotNullish(validateAddressLine(config.address.addressLine1)),
-    );
-
-    useImperativeHandle(
-      ref,
-      () => ({
-        submit: () => {
-          if (!hasSomeError) {
-            onSubmit(currentCardIndividualDeliveryConfig);
+type PropsWithAddress = Omit<Props, "address"> & { address: CompleteAddressWithContactInput };
+
+const CardWizardIndividualDeliveryWithAddress = forwardRef<
+  CardWizardIndividualDeliveryRef,
+  PropsWithAddress
+>(({ members, address, onSubmit }: PropsWithAddress, ref) => {
+  const [currentCardIndividualDeliveryConfig, setCardIndividualDeliveryConfig] =
+    useState<CardIndividualDeliveryConfig>(() => members.map(member => ({ member, address })));
+
+  const [editingAddress, setEditingAddress] = useState<[Address, number] | null>(null);
+
+  const hasSomeError = currentCardIndividualDeliveryConfig.some(config =>
+    isNotNullish(validateAddressLine(config.address.addressLine1)),
+  );
+
+  useImperativeHandle(
+    ref,
+    () => ({
+      submit: () => {
+        if (!hasSomeError) {
+          onSubmit(currentCardIndividualDeliveryConfig);
+        }
+      },
+    }),
+    [currentCardIndividualDeliveryConfig, hasSomeError, onSubmit],
+  );
+
+  return (
+    <View>
+      {currentCardIndividualDeliveryConfig.map((config, index) => {
+        const initials =
```

Comment:
Could be nice to extract this in lake `string` utils (later)

Reply:
yeah, or even have `<Avatar />` take a partial user (`{firstName?: string | null | undefined, lastName?: string | null | undefined}`), I've been meaning to do that for a while 😄